server/src/main/java/org/eclipse/lsp/cobol/service/delegates/references/ElementOccurrences.java

```diff
@@ -17,118 +17,107 @@
 import lombok.NonNull;
 import lombok.Value;
 import org.eclipse.lsp.cobol.core.model.tree.Context;
-import org.eclipse.lsp.cobol.core.semantics.outline.RangeUtils;
 import org.eclipse.lsp.cobol.service.CobolDocumentModel;
 import org.eclipse.lsp.cobol.service.delegates.validations.AnalysisResult;
 import org.eclipse.lsp4j.*;
 
 import javax.annotation.Nullable;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
 import java.util.function.Function;
 import java.util.function.Predicate;
-import java.util.function.Supplier;
 import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 import static org.eclipse.lsp.cobol.service.PredefinedCopybooks.PREF_IMPLICIT;
 import static org.eclipse.lsp.cobol.service.utils.SyntaxTreeUtil.findNodeByPosition;
 
 /**
  * This occurrences provider resolves the requests for the semantic elements based on its positions.
+ * (new Element(Collections.emptyList(), Collections.emptyList()))
```

Comment:
Did you add this line on purpose?

Reply:
I missed to remove. My bad :-D. Good catch, thanks